### PR TITLE
Added clarification for pre-subdomain isolation URLs

### PR DESCRIPTION
**Pull request by ppottorff**
The content isn't really clear that once subdomain isolation is enabled, path based access then returns 404. When we enabled this feature, we had impacted users, broken pipelines where paths (/raw/ etc...) no longer worked.

### Why:

Closes [issue link]

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by CreationNationInovation — APPROVED** [review]

**Review by CreationNationInovation — DISMISSED** [review]

**Comment by janiceilene** [comment]
@ppottorff Thanks so much for opening a PR! I'll get this triaged for review ⚡

File: content/admin/configuration/configuring-network-settings/enabling-subdomain-isolation.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ topics:
 
 Subdomain isolation mitigates cross-site scripting and other related vulnerabilities. For more information, see "[Cross-site scripting](http://en.wikipedia.org/wiki/Cross-site_scripting)" on Wikipedia. We highly recommend that you enable subdomain isolation on {% data variables.product.product_location %}.
 
-When subdomain isolation is enabled, {% data variables.product.prodname_ghe_server %} replaces several paths with subdomains.
+When subdomain isolation is enabled, {% data variables.product.prodname_ghe_server %} replaces several paths with subdomains and references to user-supplied content by path will result in a 404 error.
```
Comment (lucascosti) [inline content/admin/configuration/configuring-network-settings/enabling-subdomain-isolation.md:23]: Thanks for this contribution, @ppottorff!

I briefly tested this on a GHES instance, and it wasn't all paths that would 404. For example, images in `http(s)://HOSTNAME/media/` looks like are redirected to the new path. Content  in `http(s)://HOSTNAME/raw/` doesn't get redirects, so I've adjusted your wording to mention that some URLs might `404`.

```suggestion
When subdomain isolation is enabled, {% data variables.product.prodname_ghe_server %} replaces several paths with subdomains. After enabling subdomain isolation, attempts to access the previous paths for some user-supplied content, such as `http(s)://HOSTNAME/raw/`, may return `404` errors .
```

File: content/admin/configuration/configuring-network-settings/enabling-subdomain-isolation.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ topics:
 
 Subdomain isolation mitigates cross-site scripting and other related vulnerabilities. For more information, see "[Cross-site scripting](http://en.wikipedia.org/wiki/Cross-site_scripting)" on Wikipedia. We highly recommend that you enable subdomain isolation on {% data variables.product.product_location %}.
 
-When subdomain isolation is enabled, {% data variables.product.prodname_ghe_server %} replaces several paths with subdomains.
+When subdomain isolation is enabled, {% data variables.product.prodname_ghe_server %} replaces several paths with subdomains. After enabling subdomain isolation, attempts to access the previous paths for some user-supplied content, such as `http(s)://HOSTNAME/raw/`, may return `404` errors .
```
Comment (lucascosti) [inline content/admin/configuration/configuring-network-settings/enabling-subdomain-isolation.md:23]:
```suggestion
When subdomain isolation is enabled, {% data variables.product.prodname_ghe_server %} replaces several paths with subdomains. After enabling subdomain isolation, attempts to access the previous paths for some user-supplied content, such as `http(s)://HOSTNAME/raw/`, may return `404` errors.
```

**Review by lucascosti — APPROVED** [review]
🚀

**Pull request merged**